scripts/upgrade

```diff
@@ -12,18 +12,29 @@ domain=$(ynh_app_setting_get "$app" domain)
 path=$(ynh_app_setting_get "$app" path)
 is_public=$(ynh_app_setting_get "$app" is_public)
 
+### This helper will compare the version of the currently installed app and the version of the upstream package.
+### $upgrade_type can have 2 different values
+### - UPGRADE_APP if the upstream app version has changed
+### - UPGRADE_PACKAGE if only the YunoHost package has changed
+### ynh_check_app_version_changed will stop the upgrade if the app is up to date.
+### UPGRADE_APP should be used to upgrade the core app only if there's an upgrade to do.
+upgrade_type=$(ynh_check_app_version_changed)
+
 # Remove trailing "/" for next commands
 path=${path%/}
 
 # Retreive sources and install them
 src_path=/var/www/$app
 sudo rm -rf $src_path
```

Comment:
Did you try an upgrade?

Reply:
I did could update from inside the yunohost admin panel but it wasn't really an upgrade as it upgrade to the same cesium versium but with the yunohost version tag on it.
Version    v1.3.11~ynh1

![image](https://user-images.githubusercontent.com/464782/59372989-fa6b2b80-8d48-11e9-8905-439983719154.png)

Reply:
Ok, because it is removing the source and not reinstalling new sources because of bellow condition.

Reply:
Well and because v1.3.11 is the last release version too :-)

Reply:
Try to install this package without this change and upgrade with your change. You’ll see.

Reply:
@Thatoo, please, test your changes. This is not working.
I don’t want this to be merged.

The manual forced upgrade in case there is no upgrade does remove Cesium, but does not reinstall it.

That’s a shame.

Reply:
I'll install a yunohost in a virtualbox to do my tests for Cesium and Synapse.
Give me some time then.

Reply:
Ok, then you don’t see it, in the case upstream project doesn’t need to be upgraded.
The sources removal pointed by this comment is always executed.
Then, bellow, if no upgrade is necessary, nothing is installed.
In the end, there is nothing is into `/var/www/cesium`, and you are fucked-up, sources are no longer here.

You should move the remove into the `if [ "$upgrade_type" == "UPGRADE_APP" ]` block.

Reply:
I see, I assumed that this script would be launched only with new version but you are right. I got it and I'm changing it right now.

Reply:
Done!
I moved also the `mkdir` and the `chown` into `if [ "$upgrade_type" == "UPGRADE_APP" ]` block.